docs/docs/Components/components-processing.md

```diff
@@ -298,12 +298,43 @@ This component formats `DataFrame` or `Data` objects into text using templates,
 
 To use this component, create variables for values in the `template` the same way you would in a [Prompt](/components-prompts) component. For `DataFrames`, use column names, for example `Name: {Name}`. For `Data` objects, use `{text}`.
 
+To use the **Parser** component with a **Structured Output** component, do the following:
+
+1. Connect a **Structured Output** component's **DataFrame** output to the **Parser** component's **DataFrame** input.
+2. Connect the **File** component to the **Structured Output** component's **Message** input.
+3. Connect the **OpenAI** model component's **Language Model** output to the **Structured Output** component's **Language Model** input.
+
+The flow looks like this:
+
+![A parser component connected to OpenAI and structured output](/img/component-parser.png)
+
+4. In the **Structured Output** component, click **Open Table**.
+This opens a pane for structuring your table.
+The table contains the rows **Name**, **Description**, **Type**, and **Multiple**.
```

Comment:
Is this something the user can see for themselves? If so, I'm not sure why it's important to include this information.

Reply:
It is, but it tripped me up initially because I was looking for columns named after the columns I was parsing. So I have the component column names here to differentiate it from the loaded data.